Lesson 3

wanne27 opened the pull request:
второе задание 3 урока

wanne27 committed:
2 задание нового урока

wanne27 committed:
3 задание

поработал с FileStream, GZipStream, DirectoryInfo

Izigraim commented on Ilya Khamichenok/003_Программирование ввода-вывода/Task3/Task3/SearchFile.cs:
```diff
@@ -0,0 +1,14 @@
+﻿namespace Task3
+{
+    internal class SearchFile
```
Если создаешь класс, который будет выступать как Utils и внутри не будет хранить какого либо состояния, то делай его Static и функции тоже Static.

wanne27 committed:
сделал static class

wanne27 committed:
дополнительное задание, создать 100 папок и удалить

wanne27 committed:
4 задание

реализовал на WPF  Label, Button, ColorPicker
при выборе цвета в ColorPicker, Label меняет цвет, при нажатии кнопки сохраняет этот цвет в IsolatedStorageFile, и при запуске приложения цвет будет вытягиваться из изолированного хранилища и закрашивать Label
